pkg/api/api.go

```diff
@@ -33,4 +33,5 @@ type API struct {
 	TransformNetwork TransformNetwork    `yaml:"network" doc:"## Transform Network API\nFollowing is the supported API format for network transformations:\n"`
 	WriteLoki        WriteLoki           `yaml:"loki" doc:"## Write Loki API\nFollowing is the supported API format for writing to loki:\n"`
 	ExtractAggregate AggregateDefinition `yaml:"aggregates" doc:"## Aggregate metrics API\nFollowing is the supported API format for specifying metrics aggregations:\n"`
+	KubeEnrich       KubeEnrich          `yaml:"k8s_enrich" doc:"## Kubernetes metadata enrichment rules\nFollowing is the supported API format for specifying some network flow enrichment rules from Kubernetes metadata:\n"`
```

Comment:
Hey, we may have not talked enough, but looks like we're not on the same page here. The purpose of the previous work done by Eran during the past weeks was already to close the gap with goflow-kube's kube enrichment (keeping the current enrichment work done in FL2M) and, to my knowledge, we already reached a point where we are satisfied. The purpose of PRs [here](https://github.com/netobserv/network-observability-operator/pull/60) and [there](https://github.com/netobserv/network-observability-console-plugin/pull/78) are adapting with the slight model differences that we can live with (there's no functional regression as far as I know). So I don't think we need an alternative way to do enrichment. When these two PRs are merged, we should be able to complete the merge with features parity.
Of course if we've missed something / if the gap isn't closed as I thought, it's something to bring up

Reply:
PS: or maybe I misunderstand this PR? but it seems to me that you're reintroducing the goflow-kube's enrichment in place of the one already present here?

Reply:
Ok. I initially wanted to provide a 100% compatible alternative with the previous goflow-kube-enricher, but if you think it's not needed I can just remove this Enricher and use the previous network enricher.

Reply:
yes, that would be great. I guess there's other changes that you're doing here that are relevant regardless the kube enrichment, like `BatchMaxLen`

Reply:
@mariomac in FL2P the kube enrichment is done in network transform/ AddKubernetes operation --- as @jotak stated it should provide now very similar functionally to the original kube enrichment --- but there are two points to consider:

1. if you think that we need to move this out of `network transform` and be a stand-alone stage that handles just kubernethes ... and if you think on renaming from transforming into `enrich`.

2. Also, if there are any gaps that are missed ... we should add them in